### ISSUE TITLE
Alert is shown even no restart happened for vm-agent

### ISSUE THREAD

**Issue by eugenegoncharuk** [VictoriaMetrics/helm-charts#252]
I have a default rules setup for my cluster.
This rule always sending me alerts TooManyRestarts with the definition below

``` 
  - name: vm-health
    rules:
    - alert: TooManyRestarts
      annotations:
        description: Job {{`{{`}} $labels.job {{`}}`}} has restarted more than twice in the last 15 minutes. It might be crashlooping.
        summary: '{{`{{`}} $labels.job {{`}}`}} too many restarts (instance {{`{{`}} $labels.instance {{`}}`}})'
      expr: changes(process_start_time_seconds{job=~"victoriametrics|vmagent|vmalert"}[15m]) > 2
      labels:
        severity: critical
{{- if .Values.defaultRules.additionalRuleLabels }}
{{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
{{- end }}
```

If I check my "vm-agent" pods they haven't been restarted any time.
Maybe it's worth to rewrite the rule for a different metric?

**Comment by f41gh7** [comment]
Yes, it seems to be, that alert logic is incorrect for kubernetes workloads cc @hagen1778

**Comment by hagen1778** [comment]
@f41gh7 why do you think the logic is incorrect?

@eugenegoncharuk could you please show query expression result (grafana screenshot maybe) in the time when alert triggered? Thanks!

**Comment by 7840vz** [comment]
Same, we have false-positives as it seems:
Metric:
<img width="1779" alt="image" src="https://github.com/VictoriaMetrics/helm-charts/assets/122374011/1cbbf523-3b1b-42a3-bd61-ce553c66e566">
Alert expression, same timeline:
<img width="1777" alt="image" src="https://github.com/VictoriaMetrics/helm-charts/assets/122374011/a1e1b87d-a2a9-4da0-8dc3-c41c65206668">

**Comment by AndrewChubatiuk** [comment]
Looks like it's releated to issue https://github.com/VictoriaMetrics/VictoriaMetrics/issues/767#issuecomment-1650932203
and it was fixed in v1.97.3
